.changeset/long-tomatoes-check.md

```
@@ -0,0 +1,5 @@
+---
+"@spear-ai/tailwind-config": patch
+---
+
+Fixed ESM/CJS compatibility by replacing change-case with lodash.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)